### ISSUE TITLE
YOLO3-FSAF模块问题

### ISSUE THREAD

**Issue by KingPengcj** [xuannianz/FSAF#20]
敬爱的博主，您好，在模型训练过程中，生成器create_generators提供的数据
        grid_shapes = [input_shape // 32, input_shape // 16, input_shape // 8]
        grid_shapes = np.array(grid_shapes)
        batch_grid_shapes = np.tile(grid_shapes[None], (len(image_group), 1, 1))
对应的是图像的1/32,1/16,1/8,但是模型model所在LevelSelect的过程中，根据您提供的代码，应该是图像尺寸的1/8,1/16,1/32，这个是我理解错了吗
 for level_id in range(len(strides)):
        stride = strides[level_id]
        fh = feature_shapes[level_id][0]
        fw = feature_shapes[level_id][1]
        fa = tf.reduce_prod(feature_shapes, axis=-1)
        start_idx = tf.reduce_sum(fa[:level_id])
        end_idx = start_idx + fh * fw
        cls_pred_i = tf.reshape(cls_pred[start_idx:end_idx], (fh, fw, tf.shape(cls_pred)[-1]))
        regr_pred_i = tf.reshape(regr_pred[start_idx:end_idx], (fh, fw, tf.shape(regr_pred)[-1]))
        proj_boxes = gt_boxes / stride
